airflow/README.md

```diff
@@ -248,6 +255,42 @@ _Available for Agent versions >6.0_
 
 3. [Restart the Agent][7].
 
+<!-- xxz tab xxx -->
+<!-- xxx tab "Containerized" xxx -->
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][13] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                 |
+|----------------------|-----------------------|
+| `<INTEGRATION_NAME>` | `airflow`             |
+| `<INIT_CONFIG>`      | blank or `{}`         |
+| `<INSTANCE_CONFIG>`  | `{"url": "%%host%%"}` |
+
+##### Log collection
+
+_Available for Agent versions >6.0_
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes log collection documentation][14].
+
+| Parameter      | Value                                                 |
+|----------------|-------------------------------------------------------|
+| `<LOG_CONFIG>` | `{"source": "airflow", "service": "<YOUR_APP_NAME>"}` |
+
+##### Kubernetes
```

Comment:
Note that I have this PR open that's somewhat related to this: https://github.com/DataDog/integrations-core/pull/7789

Not sure there anything to take from there as Tips, but just mentioning since it's related.

Reply:
Thanks @AlexandreYang! Do you think it's a good idea to link to that after it is merged?

Reply:
@ruthnaebeck Yes, why not, it might be helpful to some people trying similar setup I think.